backend/core/locale/fr/LC_MESSAGES/django.po

```diff
@@ -84,6 +84,13 @@ msgstr "Tournesol - confiance"
 msgid "Tournesol - preferences"
 msgstr "Tournesol - préférences"
 
+#: core/management/commands/remind_registrations.py:24
+#: core/templates/core/no_contrib_email/_body_header.inc.html:4
+#: core/templates/core/no_contrib_email/body_no_contrib.txt:1
+#: core/templates/core/no_contrib_email/body_signup_contrib.txt:1
+msgid "Want to help responsible scientific research?"
+msgstr "Vous souhaitez aider la recherche scientifique responsable?"
```

Comment:
I'm not sure this gives a good image of the scientific research.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1884/commits/cd840f71ebb9471201b270981febab2457901fad